### PR TITLE
align crab-pre(vious) with current crab-prod

### PR DESCRIPTION
**Pull request by belforte**

### CODE REVIEW

**Comment by ddaina** [comment]
please test

File: crab-pre.spec [hunk 1]
```diff
@@ -2,11 +2,11 @@
...
 #For any other change, increment version_suffix
 ##########################################
-%define version_suffix 02
-%define crabclient_version v3.210323
+%define version_suffix 00
+%define crabclient_version v3.210825
 ### RPM cms crab-pre %{crabclient_version}.%{version_suffix}
 %define wmcore_version     1.3.6.crab6
-%define crabserver_version v3.210318
+%define crabserver_version v3.2106p11
```
Comment (ddaina) [inline crab-pre.spec]: @belforte crabserver version should be v3.210611p1

Comment (belforte) [reply crab-pre.spec]: bad copy/paste. sorry

**Comment by ddaina** [comment]
please test

**Comment by belforte** [comment]
still broken sorry. please abort test

**Comment by ddaina** [comment]
please abort

**Comment by ddaina** [comment]
please test

**Comment by smuzaffar** [comment]
+externals

**Pull request merged**